pathspider/cmd/measure.py

```diff
@@ -8,10 +8,13 @@
 
 from straight.plugin import load
 
+import pathspider.cmd.upload as upload
```

Comment:
PATHspider organises imports in source files. This should go between the pathspider.base and pathspider.network imports below.